project-base/app/src/DataFixtures/Demo/CustomerUserDataFixture.php

```diff
@@ -21,35 +21,37 @@
-    private const KEY_DELIVERY_ADDRESS = 'deliveryAddress';
+    private const string KEY_CUSTOMER_USER_DATA = 'customerUserData';
+    private const string KEY_BILLING_ADDRESS = 'billingAddress';
+    private const string KEY_DELIVERY_ADDRESS = 'deliveryAddress';
 
-    private const KEY_CUSTOMER_USER_DATA_FIRST_NAME = 'firstName';
-    private const KEY_CUSTOMER_USER_DATA_LAST_NAME = 'lastName';
-    private const KEY_CUSTOMER_USER_DATA_EMAIL = 'email';
-    private const KEY_CUSTOMER_USER_DATA_PASSWORD = 'password';
-    private const KEY_CUSTOMER_USER_DATA_TELEPHONE = 'telephone';
-    private const KEY_CUSTOMER_USER_DATA_NEWSLETTER_SUBSCRIPTION = 'newsletterSubscription';
-    private const KEY_CUSTOMER_USER_DATA_UUID = 'uuid';
+    private const string KEY_CUSTOMER_USER_DATA_FIRST_NAME = 'firstName';
+    private const string KEY_CUSTOMER_USER_DATA_LAST_NAME = 'lastName';
+    private const string KEY_CUSTOMER_USER_DATA_EMAIL = 'email';
+    private const string KEY_CUSTOMER_USER_DATA_PASSWORD = 'password';
+    private const string KEY_CUSTOMER_USER_DATA_TELEPHONE = 'telephone';
+    private const string KEY_CUSTOMER_USER_DATA_NEWSLETTER_SUBSCRIPTION = 'newsletterSubscription';
+    private const string KEY_CUSTOMER_USER_DATA_UUID = 'uuid';
 
-    private const KEY_ADDRESS_COMPANY_CUSTOMER = 'companyCustomer';
-    private const KEY_ADDRESS_COMPANY_NAME = 'companyName';
-    private const KEY_ADDRESS_COMPANY_NUMBER = 'companyNumber';
-    private const KEY_ADDRESS_COMPANY_TAX_NUMBER = 'companyTaxNumber';
-    private const KEY_ADDRESS_STREET = 'street';
-    private const KEY_ADDRESS_CITY = 'city';
-    private const KEY_ADDRESS_POSTCODE = 'postcode';
-    private const KEY_ADDRESS_COUNTRY = 'country';
-    private const KEY_ADDRESS_ADDRESS_FILLED = 'addressFilled';
-    private const KEY_ADDRESS_TELEPHONE = 'telephone';
-    private const KEY_ADDRESS_FIRST_NAME = 'firstName';
-    private const KEY_ADDRESS_LAST_NAME = 'lastName';
-    private const KEY_ADDRESS_UUID = 'uuid';
+    private const string KEY_ADDRESS_COMPANY_CUSTOMER = 'companyCustomer';
+    private const string KEY_ADDRESS_COMPANY_NAME = 'companyName';
+    private const string KEY_ADDRESS_COMPANY_NUMBER = 'companyNumber';
+    private const string KEY_ADDRESS_COMPANY_TAX_NUMBER = 'companyTaxNumber';
+    private const string KEY_ADDRESS_STREET = 'street';
+    private const string KEY_ADDRESS_CITY = 'city';
+    private const string KEY_ADDRESS_POSTCODE = 'postcode';
+    private const string KEY_ADDRESS_COUNTRY = 'country';
+    private const string KEY_ADDRESS_ADDRESS_FILLED = 'addressFilled';
+    private const string KEY_ADDRESS_TELEPHONE = 'telephone';
+    private const string KEY_ADDRESS_FIRST_NAME = 'firstName';
+    private const string KEY_ADDRESS_LAST_NAME = 'lastName';
+    private const string KEY_ADDRESS_UUID = 'uuid';
+    public const string VITEK_DELIVERY_ADDRESS_UUID = 'd5595a22-cb85-4c05-846e-8475f09229ef';
```

Comment:
Can we name this something clear for developers and not based on data?

Reply:
fair point, I hope this is better - c46473bb12